### ISSUE TITLE
V11.1 - TC preview stopped working

### ISSUE THREAD

**Issue by NirBY** [kiwitcms/Kiwi#2711]
### Description of problem
Since V11.0 preview stopped working

### Version or commit hash
V11.0

### Steps to Reproduce

1.Open preview in TC
2. Verify in FF, Chrome. Edge.

### Actual results
![firefox_ud8xL1kTZS](https://user-images.githubusercontent.com/9214815/152749184-66627827-70b3-4275-ab0a-db5f1fad68cf.gif)

**Comment by atodorov** [comment]
Preview button shows in the browser console:

```
Uncaught TypeError: marked is not a function
    markdown https://public.tenant.kiwitcms.org/static/js/simplemde_security_override.2323660b2be3.js:69
    previewRender https://public.tenant.kiwitcms.org/static/simplemde/dist/simplemde.min.fd052f831c01.js:14
    A https://public.tenant.kiwitcms.org/static/simplemde/dist/simplemde.min.fd052f831c01.js:14
    onclick https://public.tenant.kiwitcms.org/static/simplemde/dist/simplemde.min.fd052f831c01.js:15
```

**Comment by NirBY** [comment]
Still not fixed in V11.4.

### What have done:
**Upgrade used steps**:
Backup first! Then execute the commands:

cd /containers/KiwiTCMS
docker-compose down
docker-compose pull
docker-compose up -d
docker exec -it kiwi_web /Kiwi/manage.py migrate
Refer to our documentation for more details!

_Notes_:
Using mariadb:10.8.2

**Comment by NirBY** [comment]
OK, it worked. was a browser cash issue.
